benchexec/container.py

```diff
@@ -714,6 +746,16 @@ def remount_with_additional_flags(mountpoint, fstype, existing_options, mountfla
     libc.mount(None, mountpoint, None, mountflags, None)
 
 
+def escape(s):
```

Comment:
For a module-level function, the name is a little bit too generic. There are many cases where we need to escape something, and this function does not solve all of them. So let's rename it to something like `escape_overlayfs_parameters` or so.